docs/testbed/README.testbed.QosRpc.md

```diff
@@ -0,0 +1,65 @@
+# Background
+
+In order to run qos sai test, it's essential to build syncd RPC container, upload relevant artifacts to docker registry, and config test code to access docker registry.
+
+
+# Build your own syncd RPC container image
+
+## understand the build system.
+
+Familiarize yourself with the SONiC build system by reviewing [doc](https://github.com/sonic-net/sonic-buildimage/blob/master/README.md)
+
+## building the syncd RPC Container Image
+
+Follow the build steps in above documentation. Before proceeding to the "build sonic image" step, build the syncd RPC container image for your platforms using below command:
+
+```
+# Build syncd rpc conitainer iamge
+make ENABLE_SYNCD_RPC=y target/docker-syncd-${platform_rpc}-rpc.gz
```

Comment:
Could we use this CMD to build the rpc container image after proceeding "build sonic image"?